### ISSUE TITLE
Follow-up: button style & column order for new text-filter configuration UI

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#4378]
This is a follow-up to [New text format configuration UI](https://github.com/backdrop/backdrop-issues/issues/1032) to address a few design and ux issues 

1) The `Configure` button should use the drop-button styles like it does in other interfaces:

Formats:
<img width="1223" alt="Screen Shot 2020-04-16 at 1 52 17 PM" src="https://user-images.githubusercontent.com/397895/79505710-a1fc5e80-7fe9-11ea-80b1-00e7b87ab88e.png">

Field UI:
<img width="1199" alt="Screen Shot 2020-04-16 at 1 52 26 PM" src="https://user-images.githubusercontent.com/397895/79505694-9dd04100-7fe9-11ea-8c16-44b69f8079f2.png">

Comment from @quicksketch:

> Unlike other forms where the drop-button contains links, these Configure buttons are part of a multi-part form, and must be HTML <input> elements for the form to submit. We'll need to update Seven to accommodate this as theme-specific changes.

2) The Enabled column should come to the left of the Filter Name column (even though it would be adjacent to the drag/drop column) because otherwise it may be difficult for people to understand how to enable filters at first glance. 

3) Add a header to the Filter Name column (we may want to remove the column header from the order column)

**Comment by jenlampton** [comment]
Comment from @jenlampton in text filter configuration issue:

> when creating a new text format I struggled to determine how to turn on a filter, since the "Enabled" column wasn't on the left hand side of the filter name.

Comment from @quicksketch in text filter configuration issue:

> I also thought that the checkbox might be more expected in the left-hand column, similar to enabling modules or any of the views bulk operations forms (i.e. admin/content).

> Let's consider moving the checkbox for enabling as part of the follow-up we already have to style the Configure buttons over at #4378.

**Comment by jenlampton** [comment]
I'm going to take a stab at this one, self-assigning.

**Comment by jenlampton** [comment]
bumping milestone

**Comment by BWPanda** [comment]
I don't believe this issue meets the criteria for a bug-fix release milestone: https://backdropcms.org/user-guide/adding-milestones-to-issues
- Never had the milestone label
- No PR in near/ready state
